templates/2.1/registrar.j2

```diff
@@ -0,0 +1,116 @@
+
+# Durable Attestation is currently marked as an experimental feature
+# In order to enable Durable Attestation, an "adapter" for a Persistent data Store 
+# (time-series like database) needs to be specified. Some example adapters can be 
+# found under "da/examples" so, for instance 
+#      "durable_attestation_import = keylime.da.examples.redis.py"
+# could be used to interact with a Redis (Persistent data Store)
+durable_attestation_import = {{ registrar.durable_attestation_import }}
+
+# If an adapter for Durable Attestation was specified, then the URL for a Persistent Store
+# needs to be specified here. A second optional URL could be specified, for a 
+# Rekor Transparency Log. A third additional URL could be specified, pointing to a 
+# Time Stamp Authority (TSA), compatible with RFC3161. Additionally, one might need to 
+# specify a path containing certificates required by the stores or TSA. Continuing with 
+# the above example, the following values could be assigned to the parameters: 
+#      "persistent_store_url=redis://127.0.0.1:6379?db=10&password=/root/redis.auth&prefix=myda"
+#      "transparency_log_url=http://127.0.0.1:3000"
+#      "time_stamp_authority_url=http://127.0.0.1:2020"
+#      "time_stamp_authority_certs_path=~/mycerts/tsa_cert1.pem"
+persistent_store_url = {{ registrar.persistent_store_url }}
+transparency_log_url = {{ registrar.transparency_log_url }}
+time_stamp_authority_url = {{ registrar.time_stamp_authority_url }}
+time_stamp_authority_certs_path = {{ registrar.time_stamp_authority_certs_path }}
+
+# If Durable Attestation was enabled, which requires a Persistent Store URL 
+# to be specified, the two following parameters control the format and enconding
+# of the stored attestation artifacts (defaults "json" for format and "" for encoding)
+persistent_store_format = {{ registrar.persistent_store_format }}
+persistent_store_encoding = {{ registrar.persistent_store_encoding }}
+
+# If Durable Attestation was enabled with a Transparency Log URL was specified,
+# the digest algorithm for signatures is controlled by this parameter (default "sha256")
+transparency_log_sign_algo = {{ registrar.transparency_log_sign_algo }}
+
+# If Durable Attestation was enabled with a Transparency Log URL was specified,
+# a keylime administrator can specify some agent attributes (including attestation
+# artifacts, such as quotes and logs) to be signed by the registrar. The use of "all"
+# will result in the whole "package" (agent + artifacts) being signed and leaving it empty
+# will mean no signing should be done.
+signed_attributes = {{ registrar.signed_attributes }}
+
+# Require an IDevID and IAK for registration. If this option is 'True',
+# any registration that does not provide IDevID and IAK keys and certificates
+# will fail. The certificates are checked in the registration process and the 
+# AK is verified by the IAK
+require_idevid_iak = {{ registrar.require_idevid_iak }}
```

Comment:
I believe this here is old.

Reply:
Yes thanks, I have replaced this with tpm_identity